addon-registry.js

```diff
@@ -111,6 +111,7 @@ class AddonConfigurationRegistry {
     this.resultantMergedAddons = [
       ...(packageJson.addons || []),
       ...(this.voltoConfigJS.addons || []),
+      ...(process.env.ADDONS || []).split(','),
```

Comment:
A bit funky here. `[].split()` will fail. Also, what's the addon separator? The comma is already used as a separator for an addon's "profiles", like `volto-slate:minimalDefault,tableButton`. The `ADDONS` key should also consider that it's supposed to include also the additional addon loaders, not just the single package name. I propose that we use the `;` as addons separators. So it would be like `ADDONS=volto-slate:minimalDefault,tableButton;@kitconcept/volto-grid-block`

Reply:
Also, at some point we'll have to decide how to go about merging the "addon profiles". What happens to the additional addon profiles? Are they all loaded? Some of them could be incompatible with each other.